src/status_im/chat/views/api/choose_contact.cljs

```diff
@@ -15,17 +15,14 @@
 (defn- select-contact [arg-index bot-db-key {:keys [name] :as contact}]
   (let [contact (select-keys contact [:address :whisper-identity :name :photo-path :dapp?])
         name    (str/replace name (re-pattern const/arg-wrapping-char) "")]
-    (dispatch [:set-command-argument [arg-index name true]])
-    (dispatch [:set-in-bot-db {:path  [:public (keyword bot-db-key)]
-                               :value contact}])
-    (dispatch [:select-next-argument])))
+    (dispatch [:set-contact-as-command-argument bot-db-key contact arg-index name])))
```

Comment:
Does this mean all these 3 events will be replaced by this one (and more to come) events? Or will the total number of events be more?